src/hooks/usePeopleFilters.ts

```diff
@@ -0,0 +1,39 @@
+import filterByCodingLanguage from 'people/utils/filterPeople';
+import { useState } from 'react';
+import { useStores } from 'store';
+import type { Person } from 'store/main';
+import type { CodingLanguage } from 'people/utils/languageLabelStyle';
+import { useFuse } from './useFuse';
+
+export type CodingLanguageFilter = Record<CodingLanguage, boolean>;
+
+const emptyCodingLanguageFilter = {} as CodingLanguageFilter;
+
+/**
+ * Filter people list by search string and selected language
+ */
+export default function usePeopleFilters() {
+  const { main } = useStores();
+  const [codingLanguageFilter, setCodingLanguageFilter] =
+    useState<CodingLanguageFilter>(emptyCodingLanguageFilter);
+  let people = filterByCodingLanguage(
+    useFuse(main.people, ['owner_alias']).filter((p: Person) => !p.hide) || [],
+    codingLanguageFilter
+  );

```

Comment:
Something like this?
```suggestion
  main.getPeople({ langages: codingLanguageFilter });
```

Reply:
@YLeight this URL format should fix it `{host}/people?search=Yleight&languages=Node,Javascript,Golang`

Reply:
May be this reason `useFuse` was used

Reply:
As I can see, this is a problem in people-test api, as search doesn't work in any branch